code/modules/research/xenobiology/crossbreeding/selfsustaining.dm

```diff
@@ -12,6 +12,7 @@ Self-sustaining extracts:
 	name = "autoslime"
 	desc = "It resembles a normal slime extract, but seems filled with a strange, multi-colored fluid."
 	var/obj/item/slime_extract/extract
+	var/effect_desc = "Creates one to four extracts that, when used, lets you choose what reaction you want."
```

Comment:
Can change entire description to:
A self-sustaining slime extract. When used, lets you choose which reaction you want.

The autoslime is the end-product. Self-sustaining has no pre-item like burning. Also always creates 4 autoslimes.